src/dotnet-scaffolding/dotnet-scaffold-aspnet/Program.cs

```diff
@@ -31,6 +31,7 @@ public static void Main(string[] args)
             .WithDescription("Add an empty razor component to a given project")
             .WithOption(projectOption)
             .WithOption(fileNameOption)
+            .WithStep<FirstTimeTelemetrySetupStep>()
```

Comment:
My comment above about needing to add this to all scaffolders in dotnet-scaffold-aspire seems even more relevant here with how many there are...

Reply:
yep removing.